.github/workflows/tag-push.yml

```diff
@@ -45,13 +45,18 @@ jobs:
         with:
           configuration: production
 
+      - name: Zip client
+        run: |
+          (cd packages/client/dist/vernite && zip -r ../../../../client.zip .)
+
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
           draft: false
           prerelease: false
           generate_release_notes: true
           append_body: true
+          tag_name: ${{ inputs.version }}
```

Comment:
There is no input in this workflow. This parameter will be automatically read from ref. This line should be deleted.